src/Symfony/Component/Console/Tests/Helper/ProgressIndicatorTest.php

```diff
@@ -109,6 +109,40 @@ public function testCustomIndicatorValues()
         );
     }
 
+    public function testCustomFinishedIndicatorValue()
+    {
+        $bar = new ProgressIndicator($output = $this->getOutputStream(), null, 100, ['a', 'b'], '✅');
+
+        $bar->start('Starting...');
+        usleep(101000);
```

Comment:
Why do you need this `usleep`? It's better if w e can remove it as it slows down the test suite.